js/civicrm_stripe.js

```diff
@@ -180,6 +180,27 @@
           }
         }
       }
+
+      // card_number not empty validation
+      if (!$form.find("input#credit_card_number").val()) {
+        $form.find("input#credit_card_number").parent().find('.crm-error').remove();
+        $form.find("input#credit_card_number").parent().append('<span class="crm-error msg">Credit-Card Number is required.</span>');
```

Comment:
You use `$form.find("input#credit_card_number")` multiple times here. Why not save it to a var? What do you think?
`var $card_number = $form.find("input#credit_card_number");`